src/lib/device-os-version-cache.js

```diff
@@ -0,0 +1,72 @@
+const path = require('path');
+const { ensureFolder } = require('../../settings');
+const fs = require('fs-extra');
+const CACHE_FILE_NAME = '.device-os-version.cache';
```

Comment:
Since the cache file is JSON, make the extension `.json`. No need for a dot at the start to hide the file.